google/cloud/exceptions.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        return u'%d %s' % (self.code, self.message)
```

Comment:
Ok cool.  I was stumped on how to resolve this otherwise. If you have alternative ideas I would love to hear them though.